src/components/socialLogin/tests/__snapshots__/FacebookButton.test.jsx.snap

```diff
@@ -0,0 +1,3 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Facebook Button Tests should not regress 1`] = `ShallowWrapper {}`;
```

Comment:
Please omit this test and implement a better snapshot test

Reply:
@ezrogha this test seems to only pass with enzyme { shallow }